packages/foam-vscode/src/features/commands/convert-links-format-in-note.ts

```diff
@@ -0,0 +1,189 @@
+import { commands, ExtensionContext, window } from 'vscode';
+import { isMdEditor } from '../../utils';
+import { Foam } from '../../core/model/foam';
+import { FoamWorkspace } from '../../core/model/workspace';
+import { fromVsCodeUri, toVsCodeRange } from '../../utils/vsc-utils';
+import { ResourceParser } from '../../core/model/note';
+import { IMatcher } from '../../core/services/datastore';
+import { convertLinkFormat } from '../../core/janitor';
+const vscode = require('vscode'); /* cannot import workspace from above statement and not sure what happened */
```

Comment:
can you try again to import `workspace` from vscode, or use `import * from vscode` and use that?
I am not introducing the `require`

Reply:
Done.